Framework/DataObjects/src/RebinnedOutput.cpp

```diff
@@ -132,6 +137,35 @@ void RebinnedOutput::finalize(bool hasSqrdErrs) {
               std::ostream_iterator<double>(g_log.debug(), " "));
     g_log.debug() << '\n';
```

Comment:
@martyngigg Should I also parallelise this function in `RebinnedOutput`? I think I would need to remove all the logging in order to parallelise it - but it seems like we don't really need the debug information any more anyway.

Reply:
@mducle I think I would prefer to remove the logging at this level. I think anyone looking to debug it in the future would be perfectly capable of putting the prints back in temporarily. This should improve the speed to as the `std::copy` will happen regardless of the log level - it will just not be displayed!